handbook/communication/style_guide.md

```diff
@@ -41,15 +41,15 @@ Assume the reader is a busy non-native English speaker.
 ### Referring to the product and features
 
 - Sourcegraph: main product, prefer using this name unless you need to be more precise
-  - Self-hosted Sourcegraph instance: only if clarification between the Sourcegraph.com instance, managed instances, and on-premises instances is required
-  - Managed Sourcegraph instance: only if clarification between the Sourcegraph.com instance, self-hosted instances, and managed instances is required
-  - Sourcegraph.com: the public instance of Sourcegraph for open source code at https://sourcegraph.com
-  - Sourcegraph integrations: the general term for our integrations
-    - Sourcegraph['s] Phabricator integration
-    - Sourcegraph['s] GitHub integration
-    - Sourcegraph['s] browser extensions
-      - Sourcegraph['s] Chrome extension
-      - Sourcegraph['s] Firefox add-on
+- Self-hosted Sourcegraph instance: only if clarification between the Sourcegraph.com instance, managed instances, and on-premises instances is required
+- Managed Sourcegraph instance: only if clarification between the Sourcegraph.com instance, self-hosted instances, and managed instances is required
+- Sourcegraph.com: the public instance of Sourcegraph for open source code at https://sourcegraph.com
```

Comment:
The indentation neither helps nor hurts my understanding here.

For me the confusion was that I assumed the capitalization of `Sourcegraph.com` was incidental (allolexic) rather than intentional. Putting quotation marks around the address might have helped:

```
- "Sourcegraph.com": the public instance of Sourcegraph for open source code at https://sourcegraph.com
```

Rationale: It's unusual and unnecessary to capitalize DNS names. Of course, the DNS system is case-insensitive (at least on the query side, details of [RFC 4343](https://tools.ietf.org/html/rfc4343) notwithstanding), but it's uncommon to do so anywhere it's not otherwise needed for lexical harmony (e.g., the beginning of a sentence).

Reply:
If I had been writing this, I would probably have been even more explicit:

```
- "Sourcegraph.com" (N.B.: capitalization): …
```